sdk/communication/Azure.Communication.Chat/CHANGELOG.md

```diff
@@ -1,13 +1,21 @@
 # Release History
 
-## 1.0.0-beta.4 (2021-02-08)
-# Breaking
-- Support for CreateChatThreadResult and AddChatParticipantsResult to handle partial errors in batch calls.
+## 1.0.0-beta.4 (2021-02-09)
+
+### Breaking Changes
+
+- Replaced `CommunicationUser` with `CommunicationUserIdentifier`.
+- Replaced `CommunicationUserCredential` with `CommunicationTokenCredential`.
+- Removed `Priority` field from `ChatMessage`.
+
+### Added
+
+- Added support for `CreateChatThreadResult` and `AddChatParticipantsResult` to handle partial errors in batch calls.
 - Added idempotency identifier parameter for chat creation calls.
-- Added support for readreceipts and getparticipants pagination.
-- Added new model for messages anc ontent types : Text, Html, ParticipantAdded, ParticipantRemoved, TopicUpdated 
-- Removed priority field (ChatMessage.Priority)
-- Added new model for errors (CommunicationError)
+- Added pagination support for `GetReadReceipts`, `GetReadReceiptsAsync` and `GetParticipants`, `GetParticipantsAsync`.
+- Added new model for messages and content types: `Text`, `Html`, `ParticipantAdded`, `ParticipantRemoved`, `TopicUpdated`.
```

Comment:
This doesn't read well. 'ParctcipantAdded' cannot possibly be "content type"

Reply:
Unfortunately, they are all `ChatMessageTypes` the Chat SDK mixes them into one type